### PR TITLE
Service account policy UI

### PR DESCRIPTION
**Pull request by adfost**
UI for getting service account policies
Screen Shot 2022-02-04 at 11 04 33 AM
.

### CODE REVIEW

File: portal-ui/src/screens/Console/Account/ServiceAccountPolicy.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Account/ServiceAccountPolicy.tsx:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

**Review by dvaldivia — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
Do we validate if its an invalid JSON in the UI?

**Comment by harshavardhana** [comment]
Like do we react to what they pasted? - the way we are highlighting it has JSON, are we able to validate/parse it as well?

**Comment by prakashsvmx** [comment]
@adfost Please resolve the conflicts.

**Review by prakashsvmx — APPROVED** [review]
Changes look good to me. :+1: 

Few minor changes can be part of #1525

**Comment by Alevsk** [comment]
> Like do we react to what they pasted? - the way we are highlighting it has JSON, are we able to validate/parse it as well?

I think that highlight happens automatically by the `CodeMirror` library, currently theres no validation in place to check if whatever the user input its a valid json, however if the json is wrong minio will return an error and we will display that on the UI

**Review by Alevsk — DISMISSED** [review]

**Review by dvaldivia — CHANGES_REQUESTED** [review]
When I create a SA without policy, it shows me a weird policy, if the SA has no policy, we should show empty
Screen Shot 2022-02-08 at 10 37 18 AM

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM

**Review by prakashsvmx — APPROVED** [review]
LGTM.

**Pull request merged**